go.mod

```diff
@@ -1,147 +1,157 @@
 module github.com/ionos-cloud/cluster-api-provider-ionoscloud
 
-go 1.22
+go 1.22.0
+
+toolchain go1.22.2
```

Comment:
it's better to set

```
go 1.22.2
```

and omit toolchain

Reply:
Ah this was automatically added again. :angry: